features/messages/requests/MarkAllReadButton.tsx

```diff
@@ -73,15 +73,17 @@ export default function MarkAllReadButton({
       {markAll.error && (
         <Snackbar severity="error">{markAll.error.message}</Snackbar>
       )}
-      <Tooltip title={t("mark_all_read_button_text")}>
+
+      <Button variant="text" onClick={() => markAll.mutate()}>
         <IconButton
-          aria-label={t("mark_all_read_button_text")}
+          aria-label={"mark messages as read"}
           loading={markAll.isLoading}
-          onClick={() => markAll.mutate()}
+          disabled={true}
```

Comment:
```suggestion
```
We don't want the button to be permanently disabled, since the button won't ever be focusable otherwise.